server/protocols/jmap/pom.xml

```diff
@@ -45,6 +45,10 @@
         </dependency>
```

Comment:
Commit name is not OK

( MAILBOX-316 Propagate listener on JMAP layer to listen for update parent folder as 'l' right )

It's more : "MAILBOX-316 Create new JMAP listener for adding lookup right to parent mailbox "